Update controller docs to more correctly represent the current state of features (#6873)
microsoft/MixedRealityToolkit-Unity: Assets/MixedRealityToolkit.SDK/Profiles/MixedRealityControllerConfigurationProfile.md

A drop down list of supported controllers by the Mixed Reality Toolkit, namely:

* Windows Mixed Reality
  * Motion Controllers
  * HoloLens 2 articulated hands
  * HoloLens 1 hands
  * HoloLens 1 clicker
* OpenVR
  * Default (fallback) controllers
  * Oculus Touch
  * Vive Wand
  * Vive Knuckles (experimental)
* Windows Gaming
  * Gamepad (like Xbox)

### Handedness
Which hand is configured for this controller definition
> Both does not configure all controllers at this time.

### Use Default model

For this controller only, use the model for the controller direct from the SDK (where available), so that you don't have to configure a custom model. This overrides the Global Model setting.

> [!NOTE]

> > This is currently supported on both Windows Mixed Reality and OpenVR, loading any controllers those platforms provide models for through their API.

Reviewer: ```suggestion
> This is currently supported on both Windows Mixed Reality and OpenVR, loading any controllers those platforms provide models through their API.
```

> > This is currently supported on both Windows Mixed Reality and OpenVR, loading any controllers those platforms provide models for through their API.

Reviewer: ```suggestion
> This is currently supported on both Windows Mixed Reality and OpenVR, loading any controller models provided by the platform API.
```
